examples/seismic/model.py

```diff
@@ -168,21 +168,21 @@ def demo_model(preset, **kwargs):
         # By default, the top part of the domain has 1.5 km/s,
         # and the bottom part of the domain has 2.5 km/s.
         shape = kwargs.pop('shape', (101, 101))
-        spacing = kwargs.pop('spacing', tuple([10. for _ in shape]))
+        spacing = kwargs.pop('spacing', tuple([1. for _ in shape]))
         origin = kwargs.pop('origin', tuple([0. for _ in shape]))
         dtype = kwargs.pop('dtype', np.float32)
         nbpml = kwargs.pop('nbpml', 10)
-        ratio = kwargs.pop('ratio', 2)
-        vp_top = kwargs.pop('vp_top', 1.5)
-        qp_top = kwargs.pop('qp_top', 10000.)
-        vs_top = kwargs.pop('vs_top', 0. * vp_top)
-        qs_top = kwargs.pop('qs_top', 0.)
-        rho_top = kwargs.pop('rho_top', 1.)
-        vp_bottom = kwargs.pop('vp_bottom', 2.5)
-        qp_bottom = kwargs.pop('qp_bottom', 10000.)
-        vs_bottom = kwargs.pop('vs_bottom', 0. * vp_bottom)
-        qs_bottom = kwargs.pop('qs_bottom', 0.)
-        rho_bottom = kwargs.pop('qs_bottom', 2.5/1.5)
+        ratio = kwargs.pop('ratio', 3)
+        vp_top = kwargs.pop('vp_top', 1.6)
+        qp_top = kwargs.pop('qp_top', 40.)
+        vs_top = kwargs.pop('vs_top', 0.4)
```

Comment:
I would prefer this to stay `0` as one of the challenge of elastic and visco-elastic is ocena bottom that goes from acoustic vs=0 to elastic vs>0 and only work with proper staggering and this would show it well)

Reply:
A water -> soft rock -> harder rock example is covered in the tutorial. These values are for soft rock -> harder rock.